ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -236,37 +259,45 @@ def getInputSpecification(cls):
+                    \item \textit{rankNcrowdingBased} - Individuals with low rank and crowding distance are more likely to be selected to survive and 
+                    proceed to the next generation. It suppoort only multi-objective optimization problem.
+                  \end{itemize}""")
     GAparams.addSub(survivorSelection)
 
     # Fitness
     fitness = InputData.parameterInputFactory('fitness', strictMode=True,
         contentType=InputTypes.StringType,
         printPriority=108,
-        descr=r"""a subnode containing the implemented fitness functions.
-                  This includes: \begin{itemize}
-                                \item    invLinear:
-                                \[fitness = -a \times obj - b \times \sum\\_{j=1}^{nConstraint} max(0,-penalty\\_j) \].
-
-                                 \item    logistic:
-                                 \[fitness = \frac{1}{1+e^{a\times(obj-b)}}\].
-
-                                                                    \item
-          feasibleFirst:                                  \[fitness =
-          -obj   \ \ \  \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \text{for}   \ \ g\\_j(x)\geq 0 \;  \forall j\]                                  and
-          \[fitness = -obj\\_{worst} - \Sigma\\_{j=1}^{J}<g\\_j(x)>   \ \ \ \ \ \ \ \   otherwise \]
-                                 \end{itemize}.""")
+        descr=r"""a subnode containing the implemented fitness functions.You can choose one of the fitness options listed below:
+                  \begin{itemize}
+                        \item \textit{invLinear} - It assigns fitness values inversely proportional to the individual's objective function values, 
+                        prioritizing solutions with lower objective function values (i.e., minimization) for selection and reproduction. It suppoort only single-objective optimization problem.\\\\
+                        $fitness = -a \times obj_j - b \times \sum_{j=1}^{nConstraint} max(0,-penalty_j) $\\
+                        where j represents an index of objects
+                        \\
+
+                        \item \textit{logistic} - It applies a logistic function to transform raw objective function values into fitness scores.  It suppoort only single-objective optimization problem.\\\\
+                        $fitness = \frac{1}{1+e^{a\times(obj-b)}}$\\ 
+
+                        \item \textit{feasibleFirst} It prioritizes solutions that meet constraints by assigning higher fitness scores to feasible solutions, 
+                        encouraging the evolution of individuals that satisfy the problem's constraints.  It suppoort single-and multi-objective optimization problem.\\\\
+                        $fitness = \left\{\begin{matrix} -obj & g_j(x)\geq 0 \; \forall j \\ -obj_{worst}- \Sigma_{j=1}^{J}<g_j(x)> & otherwise \\ \end{matrix}\right$\\
+                  \end{itemize} """)
```

Comment:
This line is why the compilation of the manual is failing. `\end{matrix}\right$` should be `\end{matrix}\right\}$`

Reply:
Of course this will cause the multiline issue, if Josh fixed it, that would be great, but if not, the removed line fixed it by breaking it to a two single line equations. (avoid using \begin{matrix}